src/components/shared/featuredeventcard/featuredeventcard.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
```

Comment:
Would strongly recommend combining `eventcard` and `featuredeventcard`! There are only like 10 lines of diffs between the two, and some of the diffs seem like unintended inconsistencies rather than real differences (e.g. `props.gridArea` is on `Box` in one file and on `Day` in another). Or at the very least, the duplicated parts can be extracted to standalone components.

I also don't think `src/components/shared` is the best home for these components but we can fix that later. These don't really seem like shared components.

Reply:
Hey yeah, I can give a shot at combining them. I know Alex expressed interest in making it easy to switch styles via a simple prop. There were enough distinct styles I thought it made sense to separate them, but there is quite a bit of duplication.

I'll give it a shot.